packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -87,7 +88,8 @@ export interface ICheckboxTokens extends FontTokens, IForegroundColorTokens, IBa
 export interface ICheckboxSlotProps {
   root: React.PropsWithRef<IViewProps>;
   checkbox: IViewProps;
-  checkmark: ITextProps;
+  checkmark?: ITextProps;
+  checkmarkSvg?: IconProps;
```

Comment:
Would it be appropriate to make this one slot that is `ITextProps | IconProps`

Reply:
Or just call it "CheckboxIcon" instead of CheckboxSVG specifically? The former feels more descriptive to me

Reply:
Have to have separate slot names so that styling can be separated. (Combining styles may cause Invalid props supplied to one type of slot). 
`checkmarkIcon` seems reasonable, so making that change.